### DISCUSSION TITLE
Searching for a code to set video mode from NTSC to PAL

### DISCUSSION THREAD

**Discussion by Rody-star** [gopro/labs#960] [Q&A]
Hi, my rear display turned into black last week on my GoPro 10. Now I am trying to change my video mode from NTSC to PAL. In the GoPro Quik app I can set anti-flicker from 60 to 50 but the GoPro turns off and on again, but stays in 60.
In the GoPro Labs app I can't find a code or checkbox to change from NTSC to PAL / from 60 to 50.
Can anybody help to fix the issue?

**Comment by dnewman-gpsw** [comment]
There is not separate control in Labs, as it uses the frame rate.  Simple request p25 or p50 for 50Hz, and p30 or p60 etc, for 60Hz.

**Reply by Rody-star** [reply]
Thanks for the quick answer.
I can change the frame rate from 60 to 30, 120 and 240. As soon as I put p25, p50, p100 or p200 the QR code becomes accepted but the cam restart in 60.

**Reply by dnewman-gpsw** [reply]
It should not change.  Camera will remember it last setting if it is working correctly, as the frame rate is saved within the preset.  As mVp25  (mode Video, 25 fps) will save 25p into the video preset, this is then what video preset will have when the camera reboots.  All for a working camera.

**Reply by Rody-star** [reply]
Thanks for your help!
This (code mVp25) still don't change my settings to 25fps. 
Everything else can be changed via the GoPro Quik app or via codes in GoPro labs.

**Reply by dnewman-gpsw** [reply]
![image](https://github.com/gopro/labs/assets/10856084/fa2cc2a3-1dc6-476f-8c92-cf649fc111cd)
switching back with
![image](https://github.com/gopro/labs/assets/10856084/cb0b9d6f-7f32-472f-b817-774582a378ad)
This works on a fully functional Labs camera.

**Comment by NewSouthMjos** [comment]
I guess i hade same problem, that OP, but reversed - fps 24,25,50,100,200 was functioning ok, but every time I changed fps to 30/60/120/240 - cam startet to lag for 1-2 sec and then reboots, with same previous mode.

I fixed it, using !PRESET command.
